rag/app/picture.py

```
@@ -97,20 +98,29 @@ def chunk(filename, binary, tenant_id, lang, callback=None, **kwargs):
             cv_model_config = get_tenant_default_model_by_type(tenant_id, LLMType.VISION)
             cv_mdl = LLMBundle(tenant_id, model_config=cv_model_config, lang=lang)
-            with io.BytesIO() as img_binary:
-                img.save(img_binary, format="JPEG")
-                img_binary.seek(0)
-                ans = cv_mdl.describe(img_binary.read())
-            callback(0.8, "CV LLM respond: %s ..." % ans[:32])
-            txt += "\n" + ans
-            tokenize(doc, txt, eng, language=lang)
-            return attach_media_context([doc], 0, image_ctx)
+            ans = cv_mdl.describe(image_bytes)
         except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)